index.html

```diff
@@ -2,10 +2,85 @@
+    <link rel="icon" href="assets/img/icon.png"/>
   </head>
   <body>
-    <h1>Hello World</h1>
-    <p>This is a web page.</p>
+    <main>
+      <div class="container">
+        <div>
+          <nav class="gradient">
+            <ul class="nav_ul flex_container">
+              <li id="apple" class="nav_hover"><a class="nav_link white" href=""><img src="assets/img/apple_logo.png" width="40px" height="auto"/></a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">Store</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">Mac</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iPod</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iPhone</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iPad</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">iTunes</a></li>
+              <li class="nav_hover"><a class="nav_link white" href="">Support</a></li>
+              <li><input class="gradient" type="search" name="search"/></li>
+            </ul>
+          </nav>
+        </div>
+
+        <div class="content">
+          <h2>What will your verse be?</h2>
+          <p>
+            Everyone has something to share. See how people are using iPad in their unique ways to add to the world&#39;s story.
+          </p>
+          <span><a href="" class="body_link">Watch the &quot;Your Verse&quot; film and explore the stories <i id="play_icon" class="far fa-play-circle"></i></a></span>
+        </div>
+      </div>
+    </main>
+   
+    <section class="promo">
+      <div class="flex_container">
+        <figure>
+          <a href=""><img src="assets\img\promo_iphone5s.jpg" alt=""/></a>
+        </figure>
+
+        <figure>
+          <a href=""><img src="assets\img\promo_iphone5c.jpg" alt=""/></a>
+        </figure>
+
+        <figure>
+          <a href=""><img src="assets\img\promo_30_years.jpg" alt=""/></a>
+        </figure>
+
+        <figure>
+          <a href=""><img src="assets\img\promo_supplier_responsibility.jpg" alt=""/></a>
+        </figure>
+      </div>
+    </section>
+
+    <footer class="flex_container">
+      <div id="footer_div">
+        <p>
+          Shop the <span><a href="" class="foot_link">Apple Online Store</a></span> (1-800-MY-APPLE), visit an <span><a href="" class="foot_link">Apple Retail Store</a></span>, or find a <span><a href="" class="foot_link">reseller</a></span>.
+        </p>
+
+        <p>
+          Copyright &#169; 2014 Apple Inc. All rights reserved. <span><a id="terms" href="" class="foot_link">Terms of Use</span></a><span><a href="" class="foot_link">Privacy Policy</a></span>
+        </p>
+      </div>
+
+      <ul class="flex_container">
+        <li><a href="" class="foot_link">Site Map<a/></li>
```

Comment:
For the links :
inside you `<ul>
Using *display flex* for positioning links is recommended

Reply:
I don't understand this suggestion right here